pygmt/helpers/decorators.py

```diff
@@ -224,6 +226,18 @@ def fmt_docstring(module_func):
             aliases.append("- {} = {}".format(arg, alias))
         filler_text["aliases"] = "\n".join(aliases)
 
+    filler_text["table-like"] = (
+        "numpy.ndarray or "
+        "pandas.DataFrame or "
+        "xarray.Dataset"
+        # "geopandas.GeoDataFrame"
+    )
+    filler_text["table-classes"] = (
+        ":class:`numpy.ndarray`, a :class:`pandas.DataFrame`, or an \n"
+        "    :class:`xarray.Dataset` made up of 1D :class:`xarray.DataArray` \n"
+        "    data variables containing the tabular data."
```

Comment:
It is necessary to remove this period or the period after each {table-classes}.
```suggestion
        "    data variables containing the tabular data"
```